### DISCUSSION TITLE
Hide Navigation if slides < slidesPerView

### DISCUSSION THREAD

**Discussion by james0r** [nolimits4web/swiper#5944] [Q&A]
Not seeing much on this only `watchOverflow` which I believe is for a different purpose. 

Pretty simple, if only two slides are rendered and slidesPerView is set to 3 for a particular breakpoint, I want to hide navigation buttons.

Edit: I've solved my problem using Twig, but I'm still curious.

**Comment by jackkemmish** [comment]
Watch overflow adds the class `swiper-pagination-lock` to the navigation/pagination elements to which sets them to `display: none;`
